File: Linkpoint/src/main/java/com/linkpoint/render/glres/textures/GLLoadedTextTexture.java
package com.linkpoint.render.glres.textures;

import android.graphics.Bitmap;
import com.linkpoint.render.RenderContext;

public class GLLoadedTextTexture extends GLLoadedTexture {
    public final float baselineOffset;

    public GLLoadedTextTexture(RenderContext renderContext, Bitmap bitmap, float f) {
        super(renderContext, bitmap);
        this.baselineOffset = f;
    }
}

File: Linkpoint/src/main/app_broken_backup/com/linkpoint/render/glres/textures/GLLoadedTextTexture.kt
package com.linkpoint.render.glres.textures

import android.graphics.Bitmap
import com.linkpoint.render.RenderContext

class GLLoadedTextTexture(
    renderContext: RenderContext,
    bitmap: Bitmap,
    val baselineOffset: Float,
) : GLLoadedTexture(renderContext, bitmap)